### PR TITLE
test file added

### PR DESCRIPTION
**Pull request by viji-autodesk**
I have added a base test file, please review

### CODE REVIEW

**Comment by viji-autodesk** [comment]
test

**Comment by viji35** [comment]
looks ok

File: pythoncode.py
Comment (viji35) [inline pythoncode.py:1]: lgtm

Comment (viji35) [reply pythoncode.py:1]: lgtm

**Review by viji35 — APPROVED** [review]
lgtm

**Review by viji-autodesk — COMMENTED** [review]
done

**Pull request merged**